README.md

```diff
@@ -1,24 +1,30 @@
-AChecker
-========
+# AChecker
 
 AChecker is an automated accessibility checker used to evaluate the accessibility of HTML pages, and help ensure they can be accessed by all individuals, including those with disabilities, using assistive technologies to navigate the Internet.
 
 AChecker live site: http://achecker.ca
 
 What sets AChecker apart from other automated accessibility checkers?
 
-1. Reviewers can interact with the system to make decisions on potential barriers that automated checkers can not determine with certainty.
-2. Choose from a range of accessibility standards to review conformance with various international accessibility requirements.
-3. Design custom accessibility guidelines tailored specifically to your organization
-4. View existing guidelines in AChecker to see exactly what it is reviewing.
-5. Design new accessibility checks and have them added to AChecker.
+- Reviewers can interact with the system to make decisions on potential barriers that automated checkers can not determine with certainty.
+- Choose from a range of accessibility standards to review conformance with various international accessibility requirements.
+- Design custom accessibility guidelines tailored specifically to your organization
+- View existing guidelines in AChecker to see exactly what it is reviewing.
+- Design new accessibility checks and have them added to AChecker.
 
-Installation
-------------
+## Requirements
 
-1. Unzip the achecker.tar.gz file into a php enabled, web accessible directory.
-2. Open the installer at AChecker/install
-3. Follow the instruction provided by the installer.
+- PHP 7+
+- MySQL 4.1.13+
+- Composer
+
+## Installation
+
+- Clone this repository
+- Be sure to have Composer [setup on your system/server](https://getcomposer.org/download/)
+- Run `composer update` to install dependencies
```

Comment:
Using ```composer update``` or ```composer install``` is the recommended way since we will probably have different dependencies to install and they are listed in the .json file.

Reply:
What if someone decided to install composer in a local project directory instead globally and running `composer update/install` reports `cannot find composer` error?